gap/matrix.gi

```diff
@@ -626,6 +626,7 @@ end;
 #end;
 #
 RECOG.ExtractLowStuff := function(m,layer,blocks,lens,canbas)
+  #canbas ... canonical basis
```

Comment:
canonical basis of what, though?

Reply:
I don't know.

Reply:
I think I understood less then I assumed, about the variable.